code/elemental/ElementalBehatTestAdmin.php

```diff
@@ -14,4 +14,7 @@ class ElementalBehatTestAdmin extends ModelAdmin
     private static $managed_models = [
         ElementalBehatTestObject::class,
     ];
+
+    private static $required_permission_codes = 'ELEMENTAL_EDIT';
```

Comment:
There's no need for this - `LeftAndMain::providePermissions()` creates a new permission for each `ModelAdmin` subclass automagically.
The correct permission code to use in behat tests for this admin section is `CMS_ACCESS_SilverStripe\FrameworkTest\Elemental\Admin\ElementalBehatTestAdmin`

Using this means we're also indirectly testing that the `LeftAndMain::providePermissions()` and `LeftAndMain::getRequiredPermission()` logic works as expected.

Reply:
I'll leave `private static $required_permission_codes = 'ELEMENTAL_EDIT'` instead of using `CMS_ACCESS_SilverStripe\FrameworkTest\Elemental\Admin\ElementalBehatTestAdmin`, since it's more readable in the behat test

Reply:
Probably, I'll rename it to give it more obvious meaning.